lib/simple-db.js

```diff
@@ -0,0 +1,173 @@
+/**
```

Comment:
This file doesn't pass the new ESLint checks. Not sure if we should be spinning this out into a separate module or fixing it up here.